be/src/exec/pipeline/operator.cpp

```diff
@@ -100,10 +104,58 @@ void Operator::eval_runtime_bloom_filters(vectorized::Chunk* chunk) {
     if (chunk == nullptr || chunk->is_empty()) {
         return;
     }
+
     if (auto* bloom_filters = runtime_bloom_filters()) {
         _init_rf_counters(true);
         bloom_filters->evaluate(chunk, _bloom_filter_eval_context);
     }
+
+    eval_filter_null_values(chunk);
+}
+
+void Operator::eval_filter_null_values(vectorized::Chunk* chunk) {
```

Comment:
Operator::eval_filter_null_values and ExecNode::eval_filter_null_values look like almostly.
so suggest to define a common function eval_filter_null_values(vectorized::Chunk* chunk, const std::vector<SlotId>& ) , and define Operator::eval_filter_null_values and ExecNode::eval_filter_null_values on this common function.

Reply:
Good suggestions.